player/src/main/java/com/focusit/jsflight/player/configurations/ScriptsConfiguration.java

```diff
@@ -0,0 +1,275 @@
+    }
+
+    public String getShouldSkipKeyboardScript()
+    {
+        return shouldSkipKeyboardScript;
+    }
+
+    public void setShouldSkipKeyboardScript(String shouldSkipKeyboardScript)
+    {
+        this.shouldSkipKeyboardScript = shouldSkipKeyboardScript;
+    }
+
+    public String getSendSignalToProcessScript()
+    {
+        return sendSignalToProcessScript;
+    }
+
+    public void setSendSignalToProcessScript(String sendSignalToProcessScript)
+    {
+        this.sendSignalToProcessScript = sendSignalToProcessScript;
+    }
+
+    public String getGetWebDriverPidScript()
+    {
+        return getWebDriverPidScript;
+    }
+
+    public void setGetWebDriverPidScript(String getWebDriverPidScript)
+    {
+        this.getWebDriverPidScript = getWebDriverPidScript;
+    }
+
+    public String getIsAsyncRequestsCompletedScript()
+    {
+        return isAsyncRequestsCompletedScript;
+    }
+
+    public void setIsAsyncRequestsCompletedScript(String isAsyncRequestsCompletedScript)
+    {
+        this.isAsyncRequestsCompletedScript = isAsyncRequestsCompletedScript;
+    }
+
+    @Override
+    public void loadDefaults()
+    {
+        // Please, sorry me for this hell, but otherwise it would look very ugly
+        // Here we initialize all uninitialized script fields with default script from resources
+        Arrays.stream(getClass().getDeclaredFields())
+                .filter(field -> !Modifier.isStatic(field.getModifiers()) && !isFieldInitialized(field))
+                .forEach(field -> {
+                    if (!initializeField(field))
+                    {
+                        throw new RuntimeException(String.format("Can't load resource for %s", field.getName()));
+                    }
+                });
+    }
+
+    private boolean initializeField(Field field)
+    {
+        try
+        {
+            getSetterForField(field).invoke(this, readResourceAsString(getScriptPathForField(field)));
+            return true;
+        }
+        catch (Exception e)
+        {
+            return false;
+        }
+    }
+
+    private boolean isFieldInitialized(Field field)
+    {
+        try
+        {
+            return getGetterForField(field).invoke(this) != null;
+        }
+        catch (Exception e)
+        {
+            return true;
+        }
+    }
+
+    private String readResourceAsString(String resourceName) throws IOException
+    {
+        try (InputStream stream = getClass().getClassLoader().getResourceAsStream(resourceName))
+        {
+            return IOUtils.toString(stream, "UTF-8");
+        }
+    }
+
+    public void syncScripts(JMeterRecorder recorder)
+    {
+        recorder.getScriptProcessor().setScenarioProcessorScript(getScenarioProcessorScript());
+        recorder.getScriptProcessor().setStepProcessorScript(getStepProcessorScript());
+    }
+
+    @Retention(RetentionPolicy.RUNTIME)
+    private @interface DefaultFile
```

Comment:
Well, I planned, that this annotation is used only here, so there is no need to place it somewhere else. But, if it's required, of course, I can move it to another file